site/docs/components/static-list/accessibility.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+## Keyboard interactions
```

Comment:
This doc looks like it was lifted from List, I imagine here we want a guide on how consumers should manage the content within a list

Might need some more detail/expansion (speak to Jacob)

For a static list component where the list items themselves are not interactable but contain interactive elements, the primary keyboard support considerations are ensuring that the interactive elements are focusable and navigable using the Tab key. Using semantic HTML and ensuring proper labeling of interactive elements will help maintain accessibility. Additional ARIA roles and properties may be used as needed for more complex interactions.

Reply:
Might need to speak to your examples here... what accessibility considerations are needed for each type of content?

Reply:
@FYI. Asmita already reviewed this with Jacob in the[ Accessibility Specs](https://www.figma.com/design/pXXcdSsLVdsSuFgpnfj9L5/Static-list?node-id=9261-33127&t=kxCqmnL4Mw3GftB3-1)